spec/controllers/chargeback_controller_spec.rb

```diff
@@ -110,4 +111,29 @@
       expect(flash_array.first[:message]).to include("rate is assigned and cannot be deleted")
     end
   end
+
+  context "#get_cis_all" do
+    elem_objects = [:storage, :miq_enterprise, :ext_management_system, :ems_cluster, :tenant]
+    elem_objects.each do |obj|
+      let!(obj) { FactoryGirl.create(obj) }
+    end
+    elements_miq = %w(enterprise storage ext_management_system ems_cluster tenant)
+    elements_miq.each do |element|
+      classtype =
+        if element == "enterprise"
+          MiqEnterprise
+        else
+          element.classify.constantize
+        end
```

Comment:
@jrafanie @aljesusg yea, that's right, it is enough for method `get_cis_all`, there are 
basically distinguishable only cases for `MiqEnterprise` and one of other models.

```
  def get_cis_all
    @edit[:cb_assign][:cis] = {}
    classtype =
      if @edit[:new][:cbshow_typ] == "enterprise"
        MiqEnterprise
      else
        @edit[:new][:cbshow_typ].classify.constantize
      end
    classtype.all.each do |instance|
      @edit[:cb_assign][:cis][instance.id] = instance.name
    end
  end
```